spec/services/github_service_spec.rb

```diff
@@ -6,8 +6,9 @@
     github = GithubService.new
     repos = github.get_repos(token)
 
-
-    expect(repos.first[:name]).to eq("brownfield-of-dreams")
-    expect(repos.count).to eq(30)
+    
+    expect(repos).to be_a(Array)
+    expect(repos.first).to have_key(:name)
+    expect(repos.first).to have_key(:html_url)
```

Comment:
Changed tests so that we are not testing specific values, but the keys. This will hopefully prevent API data changes from causing our tests to fail.